### ISSUE TITLE
CommandBar should allow narrator to access disabled button

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#873]
While using narrator, the user should be able to know that there is a button but it is disabled. Currently CommandBar skips focus on disabled button.

**Comment by dzearing** [comment]
This is not specific to commandbar. This is normal behavior in a web browser when buttons are disabled. Screen readers have a mode to be able to switch around elements, even when they are not focusable. We intentionally do not make non interactive elements focusable, as it would clutter the focus target list and make normal scenarios difficult to navigate.
